install/deploy_final_ncn.md

```diff
@@ -493,57 +487,64 @@ it is used for Cray installation and bootstrap.
 1. Obtain access to CSI.
 
     ```bash
-    ncn-m001# mkdir -pv /mnt/livecd /mnt/rootfs /mnt/sqfs
-    ncn-m001# mount -v /metal/bootstrap/cray-pre-install-toolkit-*.iso /mnt/livecd/
-    ncn-m001# mount -v /mnt/livecd/LiveOS/squashfs.img /mnt/sqfs/
-    ncn-m001# mount -v /mnt/sqfs/LiveOS/rootfs.img /mnt/rootfs/
-    ncn-m001# cp -pv /mnt/rootfs/usr/bin/csi /tmp/csi
-    ncn-m001# /tmp/csi version
-    ncn-m001# umount -vl /mnt/sqfs /mnt/rootfs /mnt/livecd
+    ncn-m001# mkdir -pv /mnt/livecd /mnt/rootfs /mnt/sqfs && \
+              mount -v /metal/bootstrap/cray-pre-install-toolkit-*.iso /mnt/livecd/ && \
+              mount -v /mnt/livecd/LiveOS/squashfs.img /mnt/sqfs/ && \
+              mount -v /mnt/sqfs/LiveOS/rootfs.img /mnt/rootfs/ && \
+              cp -pv /mnt/rootfs/usr/bin/csi /tmp/csi && \
+              /tmp/csi version && \
+              umount -vl /mnt/sqfs /mnt/rootfs /mnt/livecd
     ```
 
+    > **NOTE** `/tmp/csi` will delete itself on the next reboot. The `/tmp` directory is `tmpfs` and runs in memory;
+    > it will not persist on restarts.
+
 1. Authenticate with the cluster.
 
     ```bash
     ncn-m001# export TOKEN=$(curl -k -s -S -d grant_type=client_credentials \
-    -d client_id=admin-client \
-    -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
-    https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+                -d client_id=admin-client \
+                -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
+                https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
     ```
 
-1. Set the wipe safeguard to allow safe net-reboots on all NCNs.
+1. Set the wipe safeguard to allow safe reboots on all NCNs.
 
     ```bash
     ncn-m001# /tmp/csi handoff bss-update-param --set metal.no-wipe=1
     ```
 
-**NOTE** `/tmp/csi` will delete itself on the next reboot. The `/tmp` directory is `tmpfs` and runs in memory;
-it will not persist on restarts.
-
 <a name="remove-the-default-ntp-pool"></a>
 
-## 6. Remove the default NTP pool
+## 6. Remove the Default NTP Pool
 
-Run the following commands on `ncn-m001` to remove the default pool, which can cause contention issues with NTP.
+Run the following command on `ncn-m001` to remove the default pool, which can cause contention issues with NTP.
 
 ```bash
 ncn-m001# sed -i "s/^! pool pool\.ntp\.org.*//" /etc/chrony.conf
 ```
 
 <a name="configure-dns-and-ntp-on-each-bmc"></a>
 
-## 7. Configure DNS and NTP on each BMC
+## 7. Configure DNS and NTP on Each BMC
```

Comment:
> Is this the rule for titles, which is something like capitalize every significant word but not simple connecting words?

Yeah. I think I assumed "each" qualified as significant, but now that you mention it, I'm not confident about that. I'm gonna see if I can find a semi-definitive guide for this.

Reply:
> > Is this the rule for titles, which is something like capitalize every significant word but not simple connecting words?
> 
> Yeah. I think I assumed "each" qualified as significant, but now that you mention it, I'm not confident about that. I'm gonna see if I can find a semi-definitive guide for this.

Actually, looking in the [HPE writing style guide](https://brandcentral.hpe.com/brand-central/content/writing-style), I learn that headings like this should actually use "sentence case" -- that is, capitalize the first word and otherwise just proper nouns. So several of my edits in this PR are wrong. I'll change them accordingly. Thanks for bringing this up, Pete!